dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -76,24 +72,18 @@ public void provisionOutputFiles(String stdout, String stderr, String wdlOutputT
     }
 
     protected void addFilesToWorkflowAttachment(List<File> workflowAttachment, File zippedEntry, File tempDir) {
-        try {
-            SwaggerUtility.unzipFile(zippedEntry, tempDir);
-        } catch (IOException e) {
-            exceptionMessage(e, "Unable to get workflow attachment files from zip file " + zippedEntry.getName()
-                    + " Request not sent.", IO_ERROR);
+        if (zippedEntry != null) {
```

Comment:
What if zippedEntry is null? You just continue on; is that the intent?

Reply:
Yes, if by some chance no descriptor files or input json were obtained, then the workflow attachment will be empty, and the wes command will be send with an empty workflow attachment. However I suppose there should always be at least the primary descriptor and input json.